index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>SALAMCIK</title>
+    <link href="oformlenie.css"
+    rel="stylesheet">

```

Comment:
Мелочь, но выглядит приятнее

```suggestion
    <link href="oformlenie.css" rel="stylesheet">
```

Reply:
А ещё закрывай тег, если он сольный, это помогает лучше ориентироваться в коде 
```suggestion
    <link href="oformlenie.css" rel="stylesheet" />

```